Assets/MixedRealityToolkit/Utilities/Physics/TransformHelper.cs

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Physics
+{
+    /// <summary>
+    /// Helper class containing move/rotate/scale related utility functions.
+    /// <summary>
+    public class TransformHelper : MonoBehaviour
+    {
+        #region Properties
+
+        private Transform targetTransform;
+        
+        private Vector3 initialScale;
+        private Vector3 minimumScale;
+        private Vector3 maximumScale;
+
+        public float ScaleMinimum => minimumScale.x;
+        public float ScaleMaximum => maximumScale.x;
+
+        #endregion Properties
+
+        public void Initialize(Transform transform)
+        {
+            if (targetTransform == null)
+            {
+                targetTransform = transform;
+                initialScale = transform.localScale;
+            }
+        }
+
+        #region Scale Utilities
+
+        public void SetScaleLimits(float min, float max, bool relativeToInitialState = true)
```

Comment:
>public void SetScaleLimits(float min, float max, bool relativeToInitialState = true) [](start = 8, length = 84)

Nit: Missing doccomments - probably just take the comments from:

/// <summary>
        /// Sets the minimum/maximum scale for the transform helper.
        /// </summary>
        /// <param name="min">Minimum scale</param>
        /// <param name="max">Maximum scale</param>
        /// <param name="relativeToInitialState">If true the values will be multiplied by scale of target at startup. If false they will be in absolute local scale.</param>